From ed3c55fa6260ad125a32f9ac67f9e44a9bce7cf1 Mon Sep 17 00:00:00 2001
From: Prabhu Sundararaj <prabhu.sundarara@nxp.com>
Date: Mon, 11 May 2020 19:12:51 -0500
Subject: [PATCH] MGS-5186 Per Specification EGL_NATIVE_PIXMAP_KHR requires
 EGL_NO_CONTEXT

https://www.khronos.org/registry/EGL/extensions/KHR/EGL_KHR_image_pixmap.txt

Per Specification EGL_NATIVE_PIXMAP_KHR requires EGL_NO_CONTEXT.
So passing context will result in error.

Signed-off-by: Prabhu Sundararaj <prabhu.sundarara@nxp.com>
---
 hw/xwayland/xwayland-glamor-gbm.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: xorg-server-1.20.5/hw/xwayland/xwayland-glamor-gbm.c
===================================================================
--- xorg-server-1.20.5.orig/hw/xwayland/xwayland-glamor-gbm.c
+++ xorg-server-1.20.5/hw/xwayland/xwayland-glamor-gbm.c
@@ -166,7 +166,7 @@ xwl_glamor_gbm_create_pixmap_for_bo(Scre
     xwl_pixmap->bo = bo;
     xwl_pixmap->buffer = NULL;
     xwl_pixmap->image = eglCreateImageKHR(xwl_screen->egl_display,
-                                          xwl_screen->egl_context,
+                                          EGL_NO_CONTEXT /*xwl_screen->egl_context*/,
                                           EGL_NATIVE_PIXMAP_KHR,
                                           xwl_pixmap->bo, NULL);
 
